samples/videoToSequence/videotosequence.cpp

```diff
@@ -200,6 +200,7 @@ void DecProc(RocVideoDecoder *p_dec, VideoDemuxer *demuxer, int *pn_frame, doubl
                 seq_output_file_name = p_output_file_name[num_seq];
             }
             p_dec->ResetSaveFrameToFile();
+            // needed to flush the frames of the before the start of a new sequence by passing EOS to parser
```

Comment:
I think there is something missing here.  Please change something like below
"needed to flush last sequence frames before the decoding next sequence by passing EOS to parser"